comp/venue/organise-screens.yaml

```
@@ -12,22 +12,26 @@ description: >-
   SR owns 6 matching screens, without stands, which are used for the
   arena displays. These have the required cables to plug into HDMI sources.
 
+
   We also have some 19" screens with stands and a variety of other input types.
 
+
   We need to figure out what screens we need for the competition and
   whether we have enough of the right type.
 
+
   For the past few years we have used:
    - 5 screens at the arena (4 corners & staging).
-   - 2/3 screens for the livestream.
+   - 2-3 screens for the livestream.
    - 1 screen for scoring entry and comp ops.
    - Installed screens at the venue presenting at least 1 outside screen and 1 livestream per area of the venue.
 
   For each screen we will need something to drive it.
-  For the livestream and scoring entry screens these are people's laptops.
+  For the livestream and scoring entry screens these are often laptops that people provide, so have an inbuilt display.
```

Nit: livestreaming is rarely a laptop

```suggestion
  For the livestream and scoring entry screens these are often provided by volunteers, so don't need accounting for.
```